src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -887,9 +958,15 @@ protected override void Go()
                         continue;
 
                     case RegexCode.Set:
-                        if (Forwardchars() < 1 || !RegexCharClass.CharInClass(Forwardcharnext(), _code.Strings[Operand(0)]))
+                        if (Forwardchars() < 1)
                             break;
 
+                        {
```

Comment:
(nit) is there an advantage to using this block? When I first read this code, it looked like this brace goes with the `if` above it.

Reply:
Variables declared in a switch aren't scoped to just that case but to the whole switch.  So I'm using the extra braces to contain the scope of this `operand` local so it doesn't leak into the other cases.